widget/form.go

```diff
@@ -128,16 +132,50 @@ func (f *Form) checkValidation(err error) {
 	f.submitButton.Enable()
 }
 
+func (f *Form) setUpHints(item *FormItem) fyne.CanvasObject {
```

Comment:
I'm not sure `setUpHints` fully conveys what this function does. It is returning either the form item widget, or a vertical box containing the form item widget and hint text. Essentially returning the input side of the form item. Perhaps `createInput` to match `createLabel` above?